service/http.proxy/src/main/java/org/kaazing/gateway/service/http/proxy/HttpProxyService.java

```diff
@@ -18,11 +18,15 @@
 import static java.lang.String.format;
 
 import org.kaazing.gateway.service.ServiceContext;
+import org.kaazing.gateway.service.ServiceProperties;
 import org.kaazing.gateway.service.proxy.AbstractProxyService;
 
 import java.net.URI;
 import java.util.Collection;
+import java.util.HashMap;
```

Comment:
nit, hmm imports change without code that reflects that.  Maybe they are unused now